rpc/core/pipe.go

```diff
@@ -71,6 +72,8 @@ var (
 	mempool          *mempl.Mempool
 
 	logger log.Logger
+
+	config cfg.RPCConfig
```

Comment:
Orthogonal to this PR but why don't we wrap these properties in a struct instead of global vars?

Reply:
I don't know, but we should definitely try to keep number of global vars as min as possible. Refs https://github.com/tendermint/tendermint/issues/3367